src/metpy/plots/declarative.py

```diff
@@ -494,6 +494,16 @@ def lookup_map_feature(feature_name):
     return feat.with_scale(scaler)
 
 
+def drop_traitlets_dir(self):
+    """Exclude members inherited from HasTraits and '_'/'__' members for dir()."""
+    traits_dir = dir(HasTraits)
+    class_dir = dir(type(self))
+    return [name for name in class_dir if name not in traits_dir if not name.startswith('_')]
```

Comment:
```suggestion
    return filter(lambda name: not (name in traits_dir or name.startswith('_')), class_dir)
```
Does the filter read better? I'm honestly not sure how the original double `if` was working. 🤨

Reply:
I think the filter or some other sensible separation does read better than the double `if`